resources/java/LICENSE

```diff
@@ -0,0 +1,201 @@
+                            Apache License
```

Comment:
I think all resources in this directory be in `toolkit`, not `artman`: I'd consider this part of the gRPC package metadata generation.